sycl/include/CL/sycl/detail/pi.h

```diff
@@ -346,7 +346,8 @@ typedef enum {
   PI_KERNEL_GROUP_INFO_LOCAL_MEM_SIZE = CL_KERNEL_LOCAL_MEM_SIZE,
   PI_KERNEL_GROUP_INFO_PREFERRED_WORK_GROUP_SIZE_MULTIPLE =
       CL_KERNEL_PREFERRED_WORK_GROUP_SIZE_MULTIPLE,
-  PI_KERNEL_GROUP_INFO_PRIVATE_MEM_SIZE = CL_KERNEL_PRIVATE_MEM_SIZE
+  PI_KERNEL_GROUP_INFO_PRIVATE_MEM_SIZE = CL_KERNEL_PRIVATE_MEM_SIZE,
+  PI_KERNEL_GROUP_INFO_NUM_REGS = 0x10112
```

Comment:
Since this one doesn't have analog in OpenCL it needs to be documented here: what exactly is being queried/returned. 
Also let's follow https://github.com/intel/llvm/blob/b3977b5776477ee1fdb3bcc51b6c6e6e820c54b3/sycl/include/CL/sycl/detail/pi.h#L115 and add new values from the "top" of the values set to avoid accidental overlap.

Reply:
I've added a comment to explain what is being returned. Could you explain a little more what you mean about adding new values from the top? I thought I had done this - the largest value I found was https://github.com/intel/llvm/blob/cd486225df6bad45eed6c8b75d8f0eab75c0eddb/sycl/include/CL/sycl/detail/pi.h#L299 so I added one after that. Should I have left more of a gap?